packages/gatsby-design-tokens/package.json

```diff
@@ -2,27 +2,30 @@
   "name": "gatsby-design-tokens",
   "version": "1.0.6",
   "description": "Gatsby Design Tokens",
-  "main": "index.js",
+  "main": "dist/index.js",
+  "module": "dist/index.esm.js",
+  "source": "src/index.js",
+  "sideEffects": "false",
   "author": "Florian Kissling <flo@gatsbyjs.com>",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/gatsbyjs/gatsby.git",
     "directory": "packages/gatsby-design-tokens"
   },
+  "files": [
+    "./dist/**/*"
+  ],
   "scripts": {
-    "build": "babel src --out-dir . --ignore **/__tests__",
+    "build": "microbundle build --sourcemap=false -f es,cjs && agadoo dist/index.esm.js",
```

Comment:
why no source maps?

Reply:
wasn't sure we needed it. I don't know if sourcemaps are being shipped with node_modules. Happy to enable it :joy:

Reply:
I thought we do for our browser packages, but after checking it, apparently we don't. Let's keep it as-is and maybe we can change it separately